WPMedia/fusion-news-theme-blocks
# PEN-1618 Ability to turn off bottom border in Top Table List

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
As a PageBuilder user, I want to be able to turn off the display of the bottom border under items in the Top Table List, so that I can have more control over the display of this block

## Jira Ticket
- [PEN-1618](https://arcpublishing.atlassian.net/browse/PEN-1618)

## Acceptance Criteria
Under each display size in the Top Table List (XL, L, M, S), a new boolean Custom Field is added called Show bottom border

The default value of this custom field (and the behavior if no option is selected) should be true - meaning the bottom border should continue to display as it does currently

If a customer changes the Custom Field to false, then the bottom border should not display underneath any of the stories for the given display size. (For example, if they change it to false for Small stories but keep it as true for XL, L and M, then it should only be removed under Small stories. 

Tests and documentation are updated as necessary

## Test Steps
* In Fusion-News-Theme repo, checkout master & git pull
* After checking out this feature branch in fusion-news-theme-blocks, run rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..
* Run npx fusion start -f -l in Fusion-News-Theme
* Create a page then add Top Table list
* Under each type of block XL, L you will see checkbox 'Show bottom border'

## Effect Of Changes
### Before
Bottom border are permanent for all types XL, L, M, etc.

### After
You can configure visibility of bottom border for each time XL, L, M, etc.

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
